src/components/forecasts-index.js

```diff
@@ -0,0 +1,132 @@
+import 'bootstrap/dist/css/bootstrap.min.css';
+
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { useSelector, useStore } from 'react-redux';
+import {
+  Sparklines,
+  SparklinesLine,
+  SparklinesReferenceLine,
+} from 'react-sparklines';
+import { useState } from 'react';
+import { fetchFiveDayForecast } from '../reducers/forecastSlice';
+
+/**
+ * Forecast DOM component
+ * @returns {DOM element}
+ */
+const ForecastsIndex = () => {
+  // store variable declaration and variable to track state of store
+  const store = useStore();
+  const forecasts = useSelector((state) => state);
+  // mutable variables to hold forecast data properties
+  let searchedCity = 'City Name';
+  let averageTemp = 0;
+  let averagePressure = 0;
+  let averageHumidity = 0;
+  let tempsArray = [];
+  let pressureArray = [];
+  let humidityArray = [];
+
+  // checks that returned api call data array is not empty then assigns variable values
+  if (forecasts.forecasts.forecasts.length !== 0) {
+    searchedCity = forecasts.forecasts.forecasts[0].city.name;

```

Comment:
Would recommend not to hard code an index like this, use array method to find the value you need instead